pkg/server/node.go

```diff
@@ -269,13 +269,11 @@ func bootstrapCluster(
 	}
 
 	state := &initState{
```

Comment:
here, too, my preference would be to load from disk, so that we never hand-construct an init state and run the risk of getting it wrong.